packages/cli/src/commands/commons/options.ts

```diff
@@ -0,0 +1,22 @@
+/*
+*                      Copyright 2020 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+export const VERBOSE_OPTION = {
+  name: 'verbose' as 'verbose',
+  alias: 'v',
+  required: false,
+  description: 'Output extra logs',
+  type: 'boolean' as 'boolean',
```

Comment:
this gets cast to `KeyedOption<T>` in its only usage, so there is no need to cast the values here as well
```suggestion
  name: 'verbose',
  alias: 'v',
  required: false,
  description: 'Output extra logs',
  type: 'boolean',
```

btw, unrelated to this specific file, but the folder name "commons" is a little... uncommon, the folder name should probably be "common" like we have in other packages